arches_lingo/pkg/graphs/resource_models/Concept.json

```diff
@@ -317,10 +449,18 @@
                     "description": null,
                     "graph_id": "bf73e576-4888-11ee-8a8d-11afefc4bff7",
                     "helpenabled": false,
-                    "helptext": "",
-                    "helptitle": "",
-                    "instructions": "",
-                    "name": "Hierarchy",
+                    "helptext": {
+                        "en": ""
+                    },
```

Comment:
Good news this is localized, I guess, but disconcerting this was missing before.

Reply:
Maybe something to do with the way we're serializing I18n fields now? https://github.com/archesproject/arches/pull/11962

Reply:
Yeah the node descriptions show up in the graph designer as JSON, which seems bad

Reply:
A git bisect would tell for sure

Reply:
Honestly it seems like a fix for archesproject/arches#11315 just with some downstream things to adjust